packages/epo-react-lib/src/index.ts

```diff
@@ -1,4 +1,6 @@
 export { default as GlobalStyles } from "@/styles/globalStyles";
+export { default as localeStrings } from "@/assets/locales";
```

Comment:
see my Google Doc on translation strategies, this is set up in the partial implementation where the locale strings are included in the lib but made available to the end user to import and initialize.